src/pkg/selectors/exchange.go

```diff
@@ -598,7 +598,7 @@ func (ec exchangeCategory) pathValues(p path.Path) map[categorizer]string {
 	}
 
 	return map[categorizer]string{
-		folderCat: p.Folder(),
+		folderCat: p.Folder(false),
```

Comment:
what is this used for? I'm unsure from just this snippet if it should be escaped or not

Reply:
This is the folder input value that the scopes compare against.  Off the top of my head it should be unescaped.  But it's late, and I might be forgetting things.

Reply:
mmmm I think I get what you're saying and I think you're right that it should be unescaped